include/engine/routing_algorithms/map_matching.hpp

```diff
@@ -270,10 +271,10 @@ class MapMatching final : public BasicRoutingInterface<DataFacadeT, MapMatching<
                             reverse_core_heap,
                             prev_unbroken_timestamps_list[s].phantom_node,
                             current_timestamps_list[s_prime].phantom_node,
-                            duration_uppder_bound);
+                            duration_upper_bound);
                     }
                     else
-                    {
+                    {   
```

Comment:
Can you try running `./scripts/format.sh`? There are some strange formatting changes here.